legal-api/src/legal_api/resources/v2/business/business.py

```diff
@@ -75,6 +76,10 @@ def get_businesses(identifier: str):
     business.allowable_actions = allowable_actions
 
     business_json = business.json()
+
+    could_file = get_could_files(jwt, business)
+    business_json['couldFile'] = could_file
```

Comment:
Since we are changing this now, its better to move to a separate endpoint (`'/<string:identifier>/allowable-actions'`), as this is required only in filings-ui (and api users).

Reply:
AFAIK, only Filings UI uses allowable actions (though I'd like Edit UI to use it in the future).

Reply:
I'd prefer to not move it to another endpoint, as it increases the network verbosity by requiring 2 api calls when we can currently get it in 1 (and both can be ignored with the slim param)

Reply:
The business endpoint is used in many places, which does not care about allowable actions. I agree it adds an additional api call but it reduces the latency for all other calls to business endpoint

Reply:
I am not suggesting to remove `business.allowable_actions` from the business endpoint for now, instead introduce new endpoint for the new UI and based on the release of the new UI we can depreciate `business.allowable_actions` in business endpoint.